packages/wallets/src/features/cfd/modals/MT5PasswordModal/MT5PasswordModal.tsx

```diff
@@ -128,16 +145,14 @@ const MT5PasswordModal: React.FC<TProps> = ({ marketType, platform }) => {
         tradingPasswordChangeMutateAsync,
     ]);
 
-    const sendEmailVerification = useCallback(
-        (platform: TPlatforms.All) => {
-            show(
-                <ModalStepWrapper>
-                    <SentEmailContent platform={platform} />
-                </ModalStepWrapper>
-            );
-        },
-        [show]
-    );
+    const sendEmailVerification = useCallback(() => {
+        onClickForgotPassword();
```

Comment:
just wondering about error handling / loader etc, or do we just accept that user wont' be notified that somethign went wrong? 
(would be nice to establish some patterns)